Lib/site.py

```diff
@@ -442,7 +442,7 @@ def setcopyright():
     """Set 'copyright' and 'credits' in builtins"""
     builtins.copyright = _sitebuiltins._Printer("copyright", sys.copyright)
     builtins.credits = _sitebuiltins._Printer("credits", """\
-    Thanks to CWI, CNRI, BeOpen.com, Zope Corporation and a cast of thousands
+    Thanks to CWI, CNRI, BeOpen, Zope Corporation, the Python Software Foundation and a cast of thousands
     for supporting Python development.  See www.python.org for more information.""")
```

Comment:
Reflow to 80 columns.
```suggestion
    Thanks to CWI, CNRI, BeOpen, Zope Corporation, the Python Software
    Foundation, and a cast of thousands  for supporting Python
    development.  See www.python.org for more information.""")
```